Rakefile

```
@@ -446,3 +450,43 @@ namespace :test do
     end
   end
 end
+
+namespace :dependencies do
+  task :update do
+    target_submodule = ENV['TARGET_SUBMODULE']
+    target_version = ENV['TARGET_VERSION']
+
+    local_info = Bumpsnag.get_git_info
+    target_info = Bumpsnag.get_git_info(target_submodule)
+
+    if target_submodule.nil? || target_version.nil?
+      raise 'Submodule or version targets not provided, exiting'
+      exit(1)
+    end
+
+    pp "Updating submodule: #{target_submodule} to version: #{target_version}"
+    updated = Bumpsnag.update_submodule(target_submodule, target_version)
+
+    if updated
+      target_pr = local_info[:latest_pr] + 1
+      origin_repo = 'https://github.com/bugsnag/bugsnag-unity'
+      target_repo = target_info[:origin]
+
+      changelog_message = "Update #{target_submodule} to [#{target_version}](#{target_repo}/releases/tag/#{target_version}) [##{target_pr}](#{origin_repo}/pull/#{target_pr})"
+
+      Bumpsnag.add_changelog_entry(changelog_message, 'Dependencies')
+
+      release_branch = "bumpsnag-#{target_submodule}-#{target_version}"
+
+      commit_message = "[full ci] Update #{target_submodule} to #{target_version}"
```

Does the commit message need to start with `[full_ci]` to be functional or can it go at the end? Ideally we'd bury it at the end so humans don't need to read it.

```suggestion
      commit_message = "[full ci] Update #{target_submodule} to #{target_version}"
```